OpenRA.Mods.Common/Traits/ReloadAmmoPool.cs

```diff
@@ -58,14 +58,14 @@ protected override void Created(Actor self)
 		{
 			ammoPool = self.TraitsImplementing<AmmoPool>().Single(ap => ap.Info.Name == Info.AmmoPool);
 			modifiers = self.TraitsImplementing<IReloadAmmoModifier>().ToArray();
-			remainingTicks = Info.Delay;
+			remainingTicks = Util.ApplyPercentageModifiers(Info.Delay, modifiers.Select(m => m.GetReloadAmmoModifier()));
```

Comment:
The status of conditional traits aren't guaranteed to be correct until after `Created` has been completed - this needs to be deferred to either a FrameEndTask or the first tick.